src/commands/LettuceMeetVisualizer.ts

```diff
@@ -0,0 +1,274 @@
+    }
+
+    const allPeople = pollResponses.map(response => response.user.name);
+    const bestTimes = await this.getBestTimesForBlock(boardData, timeBlock, filterByArray);
+
+    const embedFields: EmbedFieldData[] = [];
+    // Selects the top 5 best times
+    bestTimes.slice(0, 5).forEach(({ blockStartTime, availabilityData }) => {
+      const date = DateTime.fromMillis(blockStartTime);
+      const time = date.toLocaleString(DateTime.DATETIME_MED);
+      const unavailablePeople = allPeople.filter(person => !availabilityData.peopleAvailable.includes(person));
+
+      let message = `${availabilityData.numAvailable}/${allPeople.length} people available.`;
+      if (unavailablePeople.length > 0) {
+        message += ` Unavailable: ${unavailablePeople.join(', ')}`;
+      }
+
+      embedFields.push({
+        name: `⏰ ${time}`,
+        value: message,
+      });
+    });
+
+    const embed =
+      new MessageEmbed()
+        .setTitle(`${boardData.data.event.title}`)
+        .setURL(`https://lettucemeet.com/l/${boardCode}`)
+        .setDescription(`Best times for **${timeBlock} minutes**`)
+        .setColor('#47d175')
+        .addFields(embedFields);
+
+    await interaction.editReply({ embeds: [embed] });
+  }
+
+  /**
+   * @param boardData board data
+   * @param timeBlock time block in minutes
+   * @param filterBy emails or names of the users to filter by
+   * @returns best times for the given time block
+   */
+  private async getBestTimesForBlock(
+    boardData: BoardData,
+    timeBlock: number,
+    filterBy?: string[],
+  ): Promise<{ blockStartTime: number, availabilityData: AvailabilityData }[]> {
+    const availabilityMap = await this.getAvailabilityMap(boardData, filterBy);
+    const availabilityArray = Array.from(availabilityMap.entries());
+
+    const bestTimes: { blockStartTime: number, availabilityData: AvailabilityData }[] = availabilityArray
+      .map(([startTime, availabilityData]) => {
+      // Gets the minimum number of people available for each time block.
+        let minimumAvailable = availabilityData.numAvailable;
+
+        // timeBlock and LETTUCE_MEET_INTERVAL are in minutes - convert to milliseconds.
+        const end = startTime + timeBlock * 60 * 1000;
+        for (let i = startTime; i < end; i += LETTUCE_MEET_INTERVAL * 60 * 1000) {
+          const currentNumber = availabilityMap.get(i)?.numAvailable || 0;
+          minimumAvailable = Math.min(minimumAvailable, currentNumber);
+        }
+
+        // Returns the start time and the minimum number of people available.
+        return {
+          blockStartTime: startTime,
+          availabilityData: { numAvailable: minimumAvailable, peopleAvailable: availabilityData.peopleAvailable },
+        };
+      });
+
+    bestTimes.sort((a, b) => b.availabilityData.numAvailable - a.availabilityData.numAvailable);
+    return bestTimes;
+  }
+
+  /**
+   * Gets the availability map for a given board.
+   * @param boardData board data
```

Comment:
Same thing goes for comments like boardData or timeBlock: pretend like you're someone who's never seen this code before and think about what info they'd need to know to understand what this function does in a vaccum